misc/scripts/install-local.sh

```diff
@@ -371,6 +397,8 @@ if [[ $? -ne 0	 ]]; then
 	return 1
 fi
 
+makeVirtualDeb
```

Comment:
Move this before the dependencies install to keep track of unneeded packages after uninstall

Reply:
And remove the apt installs of the dependencies, since the deb install would do that for us